finite_difference/src/Makefile

```diff
@@ -116,7 +141,7 @@ tile_mod.o: region_mod.o
 subdomain_mod.o: $(PARALLEL_MOD).o region_mod.o
 
 %.o: $(ROOT)/%.[Ff]90
-	$(F90) $(F90FLAGS) -c $<
+	$(F90) $(F90FLAGS) -I${FORTCL_DIR} -c $<
```

Comment:
I actually had to look it up, but indeed in make $() and ${} are the same ... I learned something :)